src/vcpkg/export.prefab.cpp

```diff
@@ -28,8 +28,8 @@ namespace vcpkg::Export::Prefab
                 return paths;
             }
 
-            Checks::exit_with_message(
-                VCPKG_LINE_INFO, "Could not enumerate directory to find modules in \"%s\": %s\n", root, ec.message());
+            Checks::msg_exit_with_message(
+                VCPKG_LINE_INFO, msg::format(msgFindModulesFailedToEnum, msg::path = root).append_raw(ec.message()));
```

Comment:
Should this call `exit_filesystem_call_error`? I'm not sure the "to find modules" part is user meaningful.